src/hooks/useInTheatersShows.ts

```diff
@@ -0,0 +1,35 @@
+import { useEffect, useState } from 'react';
+import { getMovieInTheaters } from '../helpers';
+import { ShowData } from '../types';
+
+/**
+ * Custom hook that returns a list of in Theaters shows
+ * @param sortBy | Method to sort the shows, defaults to `rating`
+ */
+const useInTheatersShows = () => {
+    const [inTheaters, setinTheaters] = useState<ShowData[] | null>(null);
+    const [loading, setLoading] = useState(true);
+
+    useEffect(() => {
+        setLoading(true);
+        const handler = async () => {
+            const movies = await getMovieInTheaters();
+            const shows: ShowData[] = [];
+
+            if (!movies) {
+                setLoading(false);
+                return;
+            }
+
+            if (movies) shows.push(...movies);
```

Comment:
The MovieData and ShowData types don't directly map. Currently if you click on an 'In Theater' card, the navigation breaks. You will want to use `convertResultsToShowData`.

Reply:
ok @Thenlie

Reply:
@Thenlie , I just investigated and realized that the endpoint for in theaters didn't have the media_type field that is why i added the "movie" as a fallback since all media in theaters are movies. As demonstrated below.

            `<Link to={`/details/${details.media_type || 'movie'}/${details.id}`} state={details}>`

Reply:
let me know if I should commit the changes.

Reply:
Yes that is right. When you use the convert function, you can pass in the media type and it will be added to the data. https://github.com/Thenlie/Streamability/blob/main/src/helpers/showTypeUtils.ts#L53

Reply:
Let me know if that doesn't make sense and I can create an example.

Reply:
`const convertResultsForIntheaters = (
    data: MovieResults | TvResults | ShowResults
): ShowData[] | null => {
    if (!data || !data.results) return null;

    const showData = data.results.map((show) => {
        return {
            id: show.id,
            poster_path: show.poster_path,
            banner_path: show.backdrop_path,
            vote_average: show.vote_average,
            vote_count: show.vote_count,
            overview: show.overview,
            media_type: show.media_type || 'movie',
            genre_ids: show.genre_ids,
            title: (show as MovieData).title,
            release_date: (show as MovieData).release_date,
        };
    });

    return showData;
};`

I think creating a new utility function for the theaters will be the best. This works. Do you think I should push the code?

Reply:
This appears to be the same as `convertResultsToShowData`.